espnet/asr/pytorch_backend/asr.py

```diff
@@ -140,11 +142,13 @@ def update_core(self):
 
         # Compute the loss at this time step and accumulate it
         optimizer.zero_grad()  # Clear the parameter gradients
```

Comment:
Is it OK to clear gradient here?

Reply:
wow I was wrong

Reply:
Maybe this is the reason of different results when using different #ngpus #accums.